content/en/account_management/login_methods.md

```diff
@@ -17,21 +17,35 @@ The following login methods are available:
 
 ## Enabling or disabling a default login method
 
-As an organization manager you can enable or disable the default login methods for your organization. New organizations start with **Datadog Username and Password** and **Sign in with Google** enabled and configured for all organizations and users. After you configure SAML, **Sign in with SAML** is also enabled.
+As an organization manager, you can enable or disable the default login methods for your organization. New organizations start with **Datadog Username and Password** and **Sign in with Google** enabled and configured for all organizations and users. After you configure SAML, **Sign in with SAML** is also enabled.
 
 1. Navigate to [Login Methods][3].
 2. Set the **Enabled by Default** setting for each method to `On` or `Off`, according to your organization's preference or policy requirements.
 3. Confirm your selection.
 
 **Note**: You cannot disable all login methods for an organization. At least one login method must be enabled by default for your organization.
 
+## Requiring Multi-factor Authentication
+
+For enhanced security, organization managers can enforce [Multi-factor Authentication][4] (MFA) for all users in the organization that log in with an email and password.
+
+1. Navigate to [Login Methods][3].
+2. Set the **Require Multi-Factor Authentication** setting to `On` or `Off`, according to your organization's preference or policy requirements.
+3. Confirm your selection.
+
+Setting **Require Multi-Factor Authentication** to `On` has two effects:
+- Users that log in with an email and password must register a second authentication factor before accessing the organization.
+- In Login Methods, a link to **View users without MFA** appears. Click on the link to see the Users list, filtered on users without MFA.
+
+The setting to require multi-factor authentication is independent of the default login method settings. Regardless of which login methods you enable by default, enforcing MFA requires a second authentication factor for users that log in with an email and password.
+
 ## Reviewing user overrides
 
 Using overrides, you can change the available login methods for individual users. In the following example, **Sign in with Google** is Off by default in the organization, but one user has it enabled by having an override set.
 
 {{< img src="account_management/login_methods_disabled_overrides_set.png" alt="Login method disabled, with user override enabled" style="width:80%;">}}
```

Comment:
We could use the following updated screenshot that includes MFA enforcement: 

![image](https://github.com/user-attachments/assets/4cd152dd-1b77-4a15-b852-6a76960fb8a7)